obspy/io/nlloc/core.py

```diff
@@ -138,17 +137,17 @@ def read_nlloc_hyp(filename, coordinate_converter=None, picks=None, **kwargs):
     signature, version, date, time = line.rsplit(" ", 3)
     creation_time = UTCDateTime().strptime(date + time, str("%d%b%Y%Hh%Mm%S"))
 
-    # maximum likelihood origin location info line
-    line = lines["HYPOCENTER"]
-
-    x, y, z = map(float, line.split()[1:7:2])
-
     if coordinate_converter:
-        x, y, z = coordinate_converter(x, y, z)
+        # maximum likelihood origin location in km info line
+        line = lines["HYPOCENTER"]
+        x, y, z = coordinate_converter(*map(float, line.split()[1:7:2]))
 
-    # origin time info line
+    # maximum likelihood origin time and location lon lat info line
     line = lines["GEOGRAPHIC"]
 
+    if not coordinate_converter:
+        x, y, z = map(float, line.split()[8:13:2])
```

Comment:
You can just add another `line = lines["GEOGRAPHIC"]` right before the time is read, no?